Distinguish between HTTP booting stages (#4304)
theforeman/foreman-documentation: guides/common/modules/con_http-booting.adoc

[id="http-booting"]
= HTTP booting

> The term HTTP booting refers to two booting stages _UEFI HTTP Boot_ and _Boot Loader HTTP Boot_.

Reviewer: ```suggestion
The term HTTP booting refers to two booting stages: UEFI HTTP Boot and Boot Loader HTTP Boot.
```

In general, we only use italics for items that users have to replace. -> I suggest to drop the emphasis.

> UEFI HTTP booting is the process of fetching the _Network Bootstrap Program_ (NBP) by the UEFI firmware using HTTP(S) protocol.

Reviewer: ```suggestion
UEFI HTTP booting is the process of fetching the Network Bootstrap Program (NBP) by the UEFI firmware using HTTP(S) protocol.
```

The NBP is typically the boot loader such as GRUB2.

> Boot loader HTTP booting is the process of fetching installation files such as kernel and initrd by the boot loader using HTTP(S) protocol.

Reviewer: ```suggestion
Boot loader HTTP booting is the process of fetching installation files such as kernel and initrd by the boot loader over HTTP(S).
```

I am unsure if my suggestion is still technically correct; it also applies to line 8

In HTTP boot, files are transferred over HTTP instead of TFTP.
Using this method can help reduce the booting time during host provisioning.
HTTP is also more reliable for transfer of large files, such as Live images, than TFTP.

> Both can be combined independently of each other, although certain conditions must be met on the host for UEFI HTTP Boot.

Reviewer: ```suggestion
You can combine them independently of each other, although your host must meet certain conditions for UEFI HTTP Boot.
```

active voice.
